package.json

```diff
@@ -4,13 +4,13 @@
   "license": "MIT",
   "engines": {
     "node": ">=8.0.0",
-    "npm": ">=5.0.0"
```

Comment:
Please to do not yet force `npm` 6. The latest released `node` version only contains `npm` 5.